### DISCUSSION TITLE
Feel much better about Copilot Chat these days

### DISCUSSION THREAD

**Discussion by LikeDreamwalker** [community/community#66299] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

I opened this discussion because I complained about my Copilot Chat experience (https://github.com/orgs/community/discussions/63456#discussion-5509961). But just today, I found Copilot Chat started to give me accurate and confident answers. Even if I make a mistake, Copilot will tell me this answer is correct and that issues should happen elsewhere, rather than saying needless apologies in the old days.

So, I am here to say thanks to the Copilot Team, and Copilot is making my day.

**Comment by Akash1134** [comment]
Hi @LikeDreamwalker ,

Thank you for sharing the feedback. It's great to hear Copilot is making your day ❤️ 

Feel free to share any feedback with us in the future. 🙂
